front/app/containers/Admin/projects/projectFolders/components/ProjectFolderRow/FolderMoreActionsMenu.tsx

```diff
@@ -52,6 +52,10 @@ const FolderMoreActionsMenu = ({ folderId, setError }: Props) => {
     actions.push(deleteAction);
   }
 
+  if (!actions.length) {
```

Comment:
I prefer to be explicit about what we mean, but not blocking.
```suggestion
  if (actions.length === 0) {
```